components/places/sql/create_sync_triggers.sql

```diff
@@ -137,7 +137,7 @@ BEGIN
            OLD.mergedAt,
            2, -- SyncStatus::Normal
            OLD.shouldUpload)
-    ON CONFLICT(guid) DO UPDATE SET
+    ON CONFLICT(id) DO UPDATE SET
```

Comment:
So, we don't care about `guid` conflict / one can't happen?

Reply:
Yep, exactly!